### PR TITLE
Create genshincodes.is-an.app

### PR DESCRIPTION
**Pull request by gamersindo1223**
I will use this to host my Api about genshin impact promotional codes
This App will scrape a website every 30 seconds and save this as an API

### Link to Website

Link: [API](https://lime-average-parrot.cyclic.app/api/genshin-codes)

### CODE REVIEW

File: domains/genshincodes.is-an.app [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "$schema": "../schemas/domain.schema.json",
+  "description": "An Api used to give promotional genshin codes by scrapping web",
+  "domain": "is-an.app",
+  "subdomain": "genshincodes",
+  "owner": {
+    "repo": "https://github.com/gamersindo1223/genshin-codeslist",
+    "email": "mrgamers11223@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://lime-average-parrot.cyclic.app",
```
Comment (tarampampam) [inline domains/genshincodes.is-an.app:11]:
```suggestion
    "CNAME": "lime-average-parrot.cyclic.app."
```

**Review by tarampampam — CHANGES_REQUESTED** [review]
Wrong file extension. Fix it, please

**Review by gamersindo1223 — COMMENTED** [review]
Changed the json

**Comment by tarampampam** [comment]
Don't see the change

**Comment by gamersindo1223** [comment]
Fixed!

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
```shell
$ dig @1.1.1.1 +noall +answer genshincodes.is-an.app
genshincodes.is-an.app.	300	IN	A	104.21.64.101
genshincodes.is-an.app.	300	IN	A	172.67.181.22
```

**Pull request merged**